src/tools/compiletest/src/runtest.rs

```diff
@@ -2413,6 +2413,11 @@ impl<'test> TestCx<'test> {
                 })
                 .into_owned();
 
+        // Normalize thread IDs in panic messages
+        normalized = static_regex!(r"thread '(?P<name>.*?)' \(\d+\) panicked")
```

Comment:
```suggestion
        normalized = static_regex!(r"thread '(?P<name>.*?)' \((rtid )?\d+\) panicked")
```

To support proposed `rtid` output